catalog.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Перфораторы в интернет-магазине «Техномарт»</title>
+  </head>
+  <body>
+<header class="header-main">
+<p class="header-logo"><a href="index.html><img src="https://via.placeholder.com/111x18" alt="Логотип «Техномарта»" width="111" height="18"></a></p>
```

Comment:
Пропустила закрывающую кавычку, из-за этого не отображается картинка
```suggestion
<p class="header-logo"><a href="index.html"><img src="https://via.placeholder.com/111x18" alt="Логотип «Техномарта»" width="111" height="18"></a></p>
```